devices/ServoMotor/samples/Program.cs

```diff
@@ -4,15 +4,16 @@
 using System;
 using System.Device.Pwm;
 using System.Diagnostics;
+using System.Threading;
 using Iot.Device.ServoMotor;
 using nanoFramework.Hardware.Esp32;
 
 Debug.WriteLine("Hello Servo Motor!");
 
 // When using an ESP32, you have to setup the pin function then create the PWM channel
-Configuration.SetPinFunction(21, DeviceFunction.PWM1);
+Configuration.SetPinFunction(5, DeviceFunction.PWM1);
```

Comment:
Maybe add to the readme the same block that I added yesterday in BMXX80 about running on a ESP32...

Reply:
It's in the readme already!

Reply:
My plan is to review all samples and everything for a consistent copy/paste of comment in a kind of batch. Also I need to adjust all the README. Need a long evening for that.

Reply:
My bad! Missed that! 😊